code/__DEFINES/~skyrat_defines/research.dm

```diff
@@ -0,0 +1,2 @@
+#define SCIENCE_JAWS_OF_LIFE_DESIGN_ID "jawsoflife_sci"
+#define SCIENCE_DRILL_DESIGN_ID "handdrill_sci"
```

Comment:
I didn't think that research design IDs were typically done as defines.
If this is something that we do in other places in the code, could you show me?

Reply:
i cant, but its better practice regardless

why change this in two places when you can change it in one? it functions, and i can get a vouch from a tg maintainer of it just _being a better idea_ if needed

Reply:
Even if it is better, it doesn't really follow format with the other research modules present in our code or TG code.

Reply:
is that really a deal-breaker? the format isnt downright required anywhere